src/ProgressOnderwijsUtils/Data/ParameterizedSqlExtensions.cs

```diff
@@ -50,4 +50,11 @@ public static ParameterizedSql ConcatenateSql(this IEnumerable<ParameterizedSql>
         }
         return new SeveralSqlFragments(builder.ToArray()).BuildableToQuery();
     }
+
+    [Pure]
+    public static ParameterizedSql AsSqlInExpression<T>(this IEnumerable<T> set)
```

Comment:
Niet zoiets als
```suggestion
    public static ParameterizedSql AsUnrolledSqlInExpression<T>(this IEnumerable<T> set)
```

...om de hint te geven dat dit een perf issue is; en dus iets anders doet dat een table-valued-parameter?